guide/8.texi

```diff
@@ -1506,7 +1506,7 @@ When the @code{LOCALE} keyword is not specified, the returning value is negative
 @comment ** 8.1.70B NUMVAL-C                                                **
 @comment *********************************************************************
 @page
-@newsubsection{8.1.70B,NUMVAL-C-V2}
+@newsubsection{8.1.70B,NUMVAL-C-2}
```

Comment:
Do you have an idea what this "B" is? Looks like two times the identical thing, no?

Reply:
Indeed, it looks like it documents the same function twice, with a different explanation. Maybe somebody started improving the description by copying it temporarily, but never finished the job ?